portstyle.css

```diff
@@ -7,32 +7,33 @@ body {
 	margin: 0;
 }
 
+.fontStyleHead {
+	font-family: Arial, sans-serif;
+	font-size: 1.2rem;
+	color: white;
+	text-decoration: none;
+}
+
 .navigationBar {
 	display: flex;
 	flex-direction: row;
 	justify-content: center;
 	color: white;
```

Comment:
Try to use hex codes where possible #FFFFFF in the case of white

Reply:
Honestly that's just there from when I was first starting and just wanted white. Thanks for pointing this out I will change now.